tests/unit/local/lambdafn/test_remote_files.py

```diff
@@ -36,7 +36,7 @@ def test_successfully_unzip_from_uri(
         path_patch.assert_called_with("layer_zip_path")
         path_mock.unlink.assert_called()
         unzip_patch.assert_called_with("layer_zip_path", "output_zip_dir", permission=0o700)
-        os_patch.environ.get.assert_called_with("AWS_CA_BUNDLE", True)
+        os_patch.environ.get.assert_called_with("AWS_CA_BUNDLE", "True")
```

Comment:
I guess you'll have to change this test if you change the behavior of `os.environ.get` in the other file, but this is probably not it.